components/multitail/multitail.p5m

```diff
@@ -0,0 +1,49 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2015 Ivan Richwalski <ivan@seppuku.net>
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+<transform file path=usr/share/doc/$(COMPONENT_NAME)/.* -> default facet.doc true>
+
+dir  path=etc
+file etc/multitail.conf.new path=etc/multitail.conf preserve=renamenew
+dir  path=etc/multitail
+file etc/multitail/colors-example.pl path=etc/multitail/colors-example.pl
```

Comment:
Do we really need to ship example files to /etc? If we need, perhaps they should be executable? Or is it better to do like Debian, shipping them in /usr/share/doc/multitail ?